sub-projects/hub/src/app/components/admin/add-admin.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import axios from 'axios';
+import logger from 'utils/logger';
+
+import {
+  Button,
+  Form,
+  Input,
+  Header,
+  Grid,
+  Segment } from 'semantic-ui-react';
+
+
+class AddAdmin extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+  handleChange({ target }) {
+    this.setState({
+      [target.name]: target.value,
+    });
+  }
+
+  handleSubmit() {
+    axios.post('http://127.0.0.1:3000/api/users', {
+      email: this.state.email,
+    })
+      .then(() => {
+        this.setState({ email: '' });
+      })
+      .catch((error) => {
+        logger.error(error);
+      });
+  }
+
+  render() {
+    return (
+      <div className='login-form'>
+        <style>{`
+                    body > div,
```

Comment:
I know y'all said this was just a stopgap, but unless we absolutely have to have this code merged now, it would be better to find a more elegant way of doing this